script.js

```diff
@@ -81,66 +94,68 @@ function searchAddress(submitAddr) {
   };
   // if optional argument supplied, call search endpoint
   if (submitAddr === true) {
-    callPelias(search_url, params);
+    callMapzen(search_url, params);
   }
   else if (inputElement.value.length > 0) {
-    callPelias(auto_url, params);
+    callMapzen(auto_url, params);
   }
 };
 
-function callPelias(url, search_params) {
-  AJAX.request(url, search_params, function (err, results) {
-    if (err) {
-      console.log(err);
-    }
-    // if autocomplete url provided
-    if (url === auto_url) {
-      if (results && results.features) {
-        // remove all children of datalist until empty, fast way of emptying
-        while (dataListEl.firstChild) {
-          dataListEl.removeChild(dataListEl.firstChild);
-        };
-        var optionLength = 5;
-        if (results.features.length < 5) {
-          optionLength = results.features.length;
-        }
-        // Loop through first five results (or fewer) and add to options
-        // TO-DO: Figure out how to display options that aren't datalist exact matches
-        for (var i = 0; i < optionLength; ++i) {
-          var option = document.createElement('option');
-          option.value = results.features[i].properties.label;
-          option.text = results.features[i].properties.label;
-          dataListEl.appendChild(option);
-        };
+function callMapzen(url, search_params) {
+  $.ajax({
+    url: url,
+    data: search_params,
+    dataType: "json",
+    success: function(data) {
+      if (url === auto_url && data.features.length > 0) {
+        addr_matches.clear();
+        addr_matches.add(data.features.map(function(addr) {return addr.properties.label}));
       }
-    }
-    // If search url provided
-    else if (url === search_url) {
-      if (results && results.features) {
-        var pos = [results.features[0].geometry.coordinates[1], results.features[0].geometry.coordinates[0]];
-        searchNeighborhoods(pos, chi_json);
+      else if (url === search_url) {
+        if (data && data.features) {
+          var pos = [data.features[0].geometry.coordinates[1], data.features[0].geometry.coordinates[0]];
+          searchNeighborhoods(pos, chi_json);
+        }
       }
+    },
+    error: function(err) {
+      console.log(err);
```

Comment:
Since it's an error, consider using [`console.error()`](https://developer.mozilla.org/en-US/docs/Web/API/Console/error)